build-system/tasks/extension-generator/bento/index.js

```diff
@@ -100,6 +104,12 @@ async function makeBentoExtension() {
   for await (const templatePath of walkDir(TEMPLATE_DIR)) {
     const destination = destinationPath(templatePath);
 
+    // Skip CSS files if --no_css flag is set
+    if (argv.no_css && /.css$/.test(templatePath)) {
```

Comment:
nit: Maybe rename to `no_amp_css`? I feel like we need better names for the ambiguity between the component JSS vs the element CSS in general, idk :/

Reply:
Done.